expression/builtin_json_vec_test.go

```diff
@@ -30,7 +30,7 @@ var vecBuiltinJSONCases = map[string][]vecExprBenchCase{
 	ast.JSONExtract:      {},
 	ast.JSONLength:       {},
 	ast.JSONType:         {{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETJson}}},
-	ast.JSONArray:        {},
+	ast.JSONArray:        {{retEvalType: types.ETJson, childrenTypes: []types.EvalType{types.ETJson}}},
```

Comment:
Please add more test cases about many args. Only one arg is not enough.

Reply:
like this?
like this?:`{retEvalType: types.ETJson, childrenTypes: []types.EvalType{
			types.ETJson，types.ETJson
			types.ETJson，types.ETJson
			types.ETJson，types.ETJson
			types.ETJson，types.ETJson
			types.ETJson，types.ETJson
			types.ETJson，types.ETJson
			types.ETJson，types.ETJson
			types.ETJson，types.ETJson
			types.ETJson，types.ETJson
			types.ETJson，types.ETJson
		}}`

Reply:
I think that add a test about three arguments is also needed.  Rest LGTM.

Reply:
done!